### DISCUSSION TITLE
Varieties of Arabic

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#3219] [Translations]
Today we're getting 8 requests to add new Arabic languages at Transifex. :(

This is weird at first look, but I know that Arabic language has many dialects (around 30?) which can be joined into five-six groups: Maghrebi (Arabic (Algeria)?), Egyptian (Arabic (Egypt)?), Mesopotamian (Arabic (Iraq)?), Levantine (Arabic (Syria)?), Peninsular Arabic (Arabic (Saudi Arabia)?) and probably Sudanese (Arabic (Sudan)?).

Is it really necessary to add many variations of Arabic language into Stellarium? If the answer is "yes", then which variations should be added at Transifex and what we should do with the current Arabic language?

@kajaji, please share your opinion on this problem.

**Comment by 10110111** [comment]
FWIW, I have the following `ar_*` locales on my Ubuntu 20.04 after installing `language-pack-ar`:
```
$ locale -a | grep -i '^ar_..\.utf8'
ar_AE.utf8
ar_BH.utf8
ar_DZ.utf8
ar_EG.utf8
ar_IN.utf8
ar_IQ.utf8
ar_JO.utf8
ar_KW.utf8
ar_LB.utf8
ar_LY.utf8
ar_MA.utf8
ar_OM.utf8
ar_QA.utf8
ar_SA.utf8
ar_SD.utf8
ar_SS.utf8
ar_SY.utf8
ar_TN.utf8
ar_YE.utf8
```
So, yes, I think it makes sense to support so many in Stellarium as well.

**Comment by gzotti** [comment]
Yes, I have just seen that. "Standard" Arabic has not even been completed&reviewed yet, so I don't know if all variants will be filled by more than 1:1 copies. If there are noticeable differences in written terminology which require separate strings, it's OK. Else it's just bloat. @kajaji ?

**Reply by kajaji** [reply]
Sorry for the late reply. I was deep in the desert without communication.
Standard Arabic is enough. It is understood by all the population (300 - 400 million) speaking Arabic in any dialect. I did not see any application or operating system that uses other than the standard Arabic.

**Reply by alex-w** [reply]
Thanks! I’ve rejected requests
